### PR TITLE
Add Bazel support.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Shikugawa <Shikugawa@gmail.com>
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
This supersedes #11. @Shikugawa could you consent? Thanks!

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -103,3 +103,44 @@ jobs:
...
+        path: "${HOME}/.cache/bazel"
+        key: bazel
+
+    - name: Install Bazelisk
+      run: |
+        curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.5.0/bazelisk-linux-amd64"
+        echo "605c4b417654a09febc7e15c6b802195fdd1c4fe8d876145fb15f10d4fa27b5b  bazelisk-linux-amd64" | sha256sum --check
+        mkdir -p "${HOME}/bin/"
+        mv bazelisk-linux-amd64 "${HOME}/bin/bazel"
+        chmod +x "${HOME}/bin/bazel"
```
Comment (lizan) [inline .github/workflows/rust.yml:125]: Bazelisk 1.5.0 is already installed in GHA ubuntu-latest, You just need an alias. https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -103,3 +103,44 @@ jobs:
...
+
+    - name: Build
+      run: |
+        "${HOME}/bin/bazel" build //...
+
+    - name: Format (buildifier)
+      run: |
+        go get -u github.com/bazelbuild/buildtools/buildifier
```
Comment (lizan) [inline .github/workflows/rust.yml:133]: Pin to released version.

File: .github/workflows/rust.yml (hunk 1 above)
Comment (PiotrSikora) [reply .github/workflows/rust.yml]: I should be able to just call it directly, shouldn't it?

Done.

File: .github/workflows/rust.yml (hunk 2 above)
Comment (PiotrSikora) [reply .github/workflows/rust.yml]: Done.

**Comment by Shikugawa** [comment]
@PiotrSikora Looks fine. Thanks!

**Comment by Shikugawa** [comment]
@googlebot I consent.

**Pull request merged**